packages/workspace/src/browser/window-helper.ts

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (C) 2017 TypeFox and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+import { injectable } from 'inversify';
+
+/**
+ * Helper for opening new browser windows and reloading the content of existing ones.
+ */
+export const WindowHelper = Symbol('WindowHelper');
+export interface WindowHelper {
```

Comment:
Could it be named `WindowService`?
Or maybe it should be `WindowOpenHandler` that implements `OpenHandler` interface and open URIs with http(s) schemas? `helper` does not communicate responsibilities, only that there is a bunch of reusable functions.